src/api/store/testimonial.py

```diff
@@ -201,13 +201,15 @@ def rank_testimonial(self, args) -> Tuple[dict, MassEnergizeAPIError]:
     try:
       id = args.get("id", None)
       rank = args.get("rank", None)
-
-      if id and rank:
+      if id:
         testimonials = Testimonial.objects.filter(id=id)
-        testimonials.update(rank=rank)
-        return testimonials.first(), None
+        if type(rank) == int  and int(rank) is not None:
```

Comment:
if the type(rank) is an int then why are we doing int(rank) again?  are you trying to check if its a valid integer?